googlesearch/__init__.py

```diff
@@ -165,40 +166,33 @@ def get_page(url, user_agent=None, verify_ssl=True):
     :raises urllib2.URLError: An exception is raised on error.
     :raises urllib2.HTTPError: An exception is raised on error.
     """
-    if user_agent is None:
-        user_agent = USER_AGENT
+    user_agent = USER_AGENT if user_agent is None else user_agent
```

Comment:
ternary operators make more complex expressions which makes code harder to read.